pyproject.toml

```diff
@@ -5,12 +5,12 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "motrainer"
 version = "0.1.3"
-requires-python = ">=3.10, <3.12"
+requires-python = ">=3.10"
```

Comment:
Here, python version 3.12 is included. However, it is not included in GitHub action `build.yml` in the `python-version` array. Can you add it there to be able to test the build?

Reply:
Added the check in GH action, thanks!